pkg/httpstream/spdy/stream.go

```diff
@@ -0,0 +1,45 @@
+package spdy
+
+import "github.com/docker/spdystream"
+
+// stream is a wrapper around spdystream.Stream that adds some additional
+// behavior to the Close() and Reset() methods.
+type stream struct {
+	stream *spdystream.Stream
+}
+
+// GetHeader retrieves the header for the specified key.
+func (s *stream) GetHeader(key string) string {
+	return s.stream.Headers().Get(key)
+}
+
+// Read delegates to the spdystream.Stream for reading.
+func (s *stream) Read(p []byte) (n int, err error) {
+	return s.stream.Read(p)
```

Comment:
If you're going to implement these you should implement WriteTo and ReadTo which are far more efficient.